### PR TITLE
Update docs for mixed protocol Services of type: LoadBalancer

### PR DESCRIPTION
**Pull request by bridgetkromhout**
This is a pull request for docs updates as we move MixedProtocolLBService from alpha to beta.

k/k PR: https://github.com/kubernetes/kubernetes/pull/109213
enhancements issue: https://github.com/kubernetes/enhancements/issues/1435

Signed-off-by: Bridget Kromhout <bridget@kromhout.org>

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -147,6 +147,7 @@ different Kubernetes components.
 | `MemoryManager` | `true` | Beta | 1.22 | |
 | `MemoryQoS` | `false` | Alpha | 1.22 | |
 | `MixedProtocolLBService` | `false` | Alpha | 1.20 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:149]:
```suggestion
| `MixedProtocolLBService` | `false` | Alpha | 1.20 | 1.23 |
```

**Comment by sftim** [comment]
/retitle Update docs for mixed protocol Services of type: LoadBalancer

**Review by sftim — COMMENTED** [review]
Markdown LGTM

@kubernetes/sig-network-pr-reviews is this change technically correct? Anything overlooked? Please add /lgtm if it's all good.

**Pull request merged**